parking_permits/views.py

```diff
@@ -602,11 +587,8 @@ def post(self, request, format=None):
             )
             return Response({"message": "Subscription cancelled"}, status=200)
         else:
+            return invalid_response(f"Unknown subscription event type {event_type}")
             logger.error(f"Unknown subscription event type {event_type}")
```

Comment:
Is this line called? It appears after the `return` statement.

Reply:
Removed leftover line